src/stringUtils.cpp

```diff
@@ -0,0 +1,29 @@
+/*
```

Comment:
I prefer to have as few diff as possible with async-profiler upstream - so instead of moving the class, and since we only need one simple utility function, let's copy just it over to `profiler.cpp`?

Reply:
👍

Reply:
(the reason is - we keep on updating our async-profiler copy, so it's best if we keep as few diff as possible, to make updates easier)